src/pages/Detail/Detail.js

```diff
@@ -1,8 +1,24 @@
 import React, { Component } from 'react';
 
+import './Detail.scss';
+
+import Nav from '../../components/Nav/Nav';
+import Footer from '../../components/Footer/Footer';
+import DetailTop from './DetailTop';
+import DetailTab from './DetailTab';
+import DetailDescContationer from './DetailDescContationer';
+
 export class Detail extends Component {
   render() {
-    return <div></div>;
+    return (
+      <div className="detail-wrap">
+        <Nav></Nav>
+        <DetailTop></DetailTop>
+        <DetailTab></DetailTab>
+        <DetailDescContationer></DetailDescContationer>
+        <Footer></Footer>
```

Comment:
- 열고 닫는 태그를 사용하시는 것은, 그 안에 들어가는 내용물이 있을 때 입니다. 아래를 참고해주세요!
```js
<Nav>
  <div>children 으로 들어갈 내용</div>
</Nav>
```
- 작성하신 코드처럼 안에 들어가는 내용물이 없을때에는 셀프클로징 해주세요.
```suggestion
        <Nav />
        <DetailTop />
        <DetailTab />
        <DetailDescContationer / >
        <Footer />
```

Reply:
확인감사합니다 성훈님~
해당 부분 모두 셀프클로징 하고 추가로 안되어있는 부분도 확인하여 반영하였습니다!!!